### PR TITLE
Feature signup login pages #21

### PR DESCRIPTION
**Pull request by stefanleoussis**
### What this PR does:
- Adds InfoSideBanner Component that displays Welcome information beside the Signup/Login Pages
- Edits the Signup/Login to match the Mocks
- Adds Primary Language Selector to Sign Up Page and adds Primary Language as a property for a User throughout the Codebase
- Removes Text Field for Username and any Reference of Username throughout the Codebase
- Makes Pages Responsive with added InfoSiderBanner Component

### Screenshots / Videos:
Screen Shot 2021-05-20 at 1 38 46 PM

Screen Shot 2021-05-20 at 1 40 04 PM

### Any information needed to test this feature:
1. Go into the client directory `cd client`
2. Run `npm install` to install packages
3. Run `npm start` to start the client side

### CODE REVIEW

File: server/models/User.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ const mongoose = require("mongoose");
 const bcrypt = require("bcryptjs");
 
 const userSchema = new mongoose.Schema({
-  username: {
+  primaryLanguage: {
     type: String,
     required: true,
     unique: true
```
Comment (dproc96) [inline server/models/User.js:8]: You'll want to remove this unique flag

File: server/validate.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 const { check, validationResult } = require("express-validator");
 
 exports.validateRegister = [
-  check("username", "Please enter a username").not().isEmpty(),
+  check("primaryLangauge", "Please enter a Primary Language").not().isEmpty(),
```
Comment (dproc96) [inline server/validate.js:4]:
```suggestion
  check("primaryLanguage", "Please enter a Primary Language").not().isEmpty(),
```

File: client/src/components/InfoSideBanner/InfoSideBanner.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import { Box, Grid } from '@material-ui/core';
+import useStyles from './useStyles';
+import { Typography } from '@material-ui/core';
+import bubble from '../../Images/bubble.svg';
+import backgroundImage from '../../Images/9382094101e259e6d9a7825ae24b1807266e08f8.png';
+
+const InfoSideBanner = () => {
```
Comment (dproc96) [inline client/src/components/InfoSideBanner/InfoSideBanner.tsx:7]: Just a note, on mobile this is appearing above the signup and login area, check with Sina if that's something to address in this PR or later

**Review by dproc96 — COMMENTED** [review]
Noticed a couple of things, also be sure to resolve the errors in the console before making this a full PR

File: server/models/User.js (hunk 1 above)
Comment (stefanleoussis) [reply server/models/User.js:8]: Oh yea definitely, thanks for spotting that

File: client/src/components/InfoSideBanner/InfoSideBanner.tsx (hunk 3 above)
Comment (stefanleoussis) [reply client/src/components/InfoSideBanner/InfoSideBanner.tsx:7]: Ok will do

File: client/src/pages/Login/Login.tsx [hunk 4]
```diff
@@ -39,7 +41,12 @@ export default function Login(): JSX.Element {
   return (
     <Grid container component="main" className={classes.root}>
       <CssBaseline />
-      <Grid item xs={12} sm={8} md={7} elevation={6} component={Paper} square>
+      <Grid item md={5}>
+        <Hidden smDown={true}>
```
Comment (dproc96) [inline client/src/pages/Login/Login.tsx:45]: 👍 Nice

**Review by dproc96 — APPROVED** [review]
Looks good to go, nice work!

**Pull request merged**